### ISSUE TITLE
3.0 Preview Spell List: Kibbles spells present prematurely

### ISSUE THREAD

**Issue by orjanbp** [WC5E/Warcraft-5e-Conversion#844]
The Ch 6: Spells doc out in the 3.0 Playtest folder has the *Flash Freeze* and *Investiture of Ice* spells already in it. 

These two spells are out of the Kibbles' spell list, which we've been wanting to fold into the book at some point.
https://www.gmbinder.com/share/-MW5r6wBJTW_lOU6gLyU

But for now, having them there is going to cause some confusion. So should remove them until we're done picking through and settling what spells to include from this collection.

**Comment by Tyloris** [comment]
I had already removed these spells from the WIP document.

**Comment by orjanbp** [comment]
@Tyloris Possible that the latest PDF version of the chapter is just out of date from the latest WIP doc then?

**Comment by Tyloris** [comment]
Yeah. There are other spells that have been added since then, from the new priest document.

**Comment by orjanbp** [comment]
The spells from the Priest doc won't be ready for show until the Priest is done. 

So for now, this task should be resolved with a PR to the Spells document here on Github that removes those two spells, and then the document on Drive should be re-deployed without those spells.

**Comment by Tyloris** [comment]
Oddly enough, those spells are not even on the github. Not sure how they got onto the PDF. In any case, they'll be fixed in the next update. I'm just waiting on resolution for the issues #845 , #846 , #847 . As well as help from the GMBinder discord on fixing a formatting issue that popped up due to one of the site's recent updates messing up the code again.
